gem5/create_predictor.py

```diff
@@ -29,18 +30,19 @@ def create_LocalBP():
     # LOCAL BP it inherits BranchPredictor (therefore you can also set those parameters)
     pred = LocalBP()
     pred.localPredictorSize = 32
-    pred.localCtrBits = 2 
+    pred.localCtrBits = 2
     pred.BTBEntries = 256
     return pred
 
 def create_TournamentBP():
-    # TOURNAMENT BP it inherits BranchPredictor 
+    # TOURNAMENT BP it inherits BranchPredictor
     pred = TournamentBP()
     pred.localPredictorSize = 32
     pred.localHistoryTableSize = 256
     pred.globalPredictorSize = 64
     pred.choicePredictorSize = 64
-    pred.BTBEntries = 256
+    if os.getenv('IS_DOCKER') != 'true':

```

Comment:
i do not understand why docker is affecting the gem5 predictors.
This is something that should not happen